src/server/common/controller/question-page-controller.test.js

```diff
@@ -149,6 +158,30 @@ describe('QuestionPageController', () => {
       expect(state.someOtherQuestion).toBe('some-other-answer')
     })
 
+    it('should redirect to exit page even if redirect uri is set', async () => {
```

Comment:
If we moved the logic to the next page helper function we could test the redirect logic (the result) in there instead of here maybe?